CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [1.2.0] - Unreleased
+## [1.2.0] - 2018-01-23
```

Comment:
I would add notes to the changelog on the removed error checking for MLPs, and the new fallback behavior for the template_ids.